docs/template.md

```diff
@@ -16,7 +16,7 @@ Examples outputs:
 
 A template is an independent Node.js project unrelated to the `generator` repository. AsyncAPI templates are managed, released, and published separately. You can also create templates and manage templates on your own.
 
-The generator uses the official [Arborist](https://www.npmjs.com/package/@npmcli/arborist) NPM library. (This means templates do not have to be published to package managers to use them.) Arborist helps the generator fetch the template's source code and use it for the generation process. 
+The generator uses the official [Arborist](https://www.npmjs.com/package/@npmcli/arborist) NPM library. (This means templates do not have to be published to package managers to use them.) Arborist helps the generator fetch the template's source code and use it for the generation process. By default, this library pulls data from public NPM repository, but you can pass URL and credentials for private repo in **params.registry** object.   
 
```

Comment:
Minor doubt, since this is a concept doc about template, we wouldn't expect the reader to know about **params.registry** yet, right🤔?
Would it make sense to replace it with something like:

By default, this library pulls data from the public NPM repository, but you can pass URL and credentials for the private repo in the generator CLI.

Or will adding a hyperlink to **params.registry** seem more proper😀?

Doubt: @derberg @alextrs